### PR TITLE
Layout OAuth client list screen

### LINKED ISSUE #423: [Portal] OAuth client list screen
- [x] study mapping from config
- layout List:
  - [x] Detail List with 
     - Name 
     - Creation date
     - Application (type, "android" | "ios" | "web-app")
     - Client ID
   - [x] client id button with edit button and copy button (click to copy to clipboard)
   - [x] Primary Button for add client button (Link to add client page)

### PR DESCRIPTION
**Pull request by axolotl0212**
Closes #423 

# Screenshots
Screenshot 2020-09-30 at 2 51 17 PM

Screenshot 2020-09-30 at 2 52 01 PM

### CODE REVIEW

File: portal/src/locale-data/en.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
   "ScreenNav.anonymous-users": "Anonymous Users",
   "ScreenNav.single-sign-on": "Single-Sign On",
   "ScreenNav.passwords": "Passwords",
+  "ScreenNav.oauth-clients": "OAuth Clients",
```
Comment (kiootic) [inline portal/src/locale-data/en.json:52]: IIRC this should be changed to `Client Applications`.

**Review by kiootic — COMMENTED** [review]
I suppose we should use `MessageBar` instead of a custom toast component.

**Comment by axolotl0212** [comment]
> I suppose we should use `MessageBar` instead of a custom toast component.

Just show message bar on top for a few seconds?

File: portal/src/locale-data/en.json (hunk 1 above)
Comment (axolotl0212) [reply portal/src/locale-data/en.json:52]: Also need to change title of the screen?

Comment (kiootic) [reply portal/src/locale-data/en.json:52]: Yes.

**Comment by kiootic** [comment]
> Just show message bar on top for a few seconds?

Just providing a close button is enough.

**Comment by axolotl0212** [comment]
@kiootic updated to only using message bar and the displayed name of OAuth clients page

File: portal/src/Notification.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { MessageBar } from "@fluentui/react";
+
+interface NotificationProps {
+  onDismiss: () => void;
+  visible: boolean;
+  msg: string;
+}
+
+export function useNotification(): {
+  showNotification: (msg: string, delay?: number) => void;
```
Comment (kiootic) [inline portal/src/Notification.tsx:11]: `delay` parameter is no use? In fact, this hook & component seems rather trivial, so I would prefer to just include it in the parent screen.

Comment (axolotl0212) [reply portal/src/Notification.tsx]: Moved to parent screen

**Review by kiootic — APPROVED** [review]

**Pull request merged**